.github/actions/setup-poetry/action.yml

```diff
@@ -29,9 +33,9 @@ runs:
       uses: actions/cache@v3
       with:
         path: .venv
-        key: venv-${{ inputs.python-version }}-${{ hashFiles('**/poetry.lock') }}-${{ inputs.groups }}
```

Comment:
For reviewers, I'm removing this from the cache key because it will contain a "," in some cases, which will crash the CI.